From 3bfaefd5eff46e0aa6e303078587bca864f93d43 Mon Sep 17 00:00:00 2001
From: Nicolas Fella <nicolas.fella@gmx.de>
Date: Sun, 6 Mar 2022 01:53:45 +0100
Subject: [PATCH] [appstream] Add content rating

I chose mild cartoon violence since it's about explosive weapons, but not in any way combined with characters
---
 data/org.kde.kmines.appdata.xml | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/data/org.kde.kmines.appdata.xml b/data/org.kde.kmines.appdata.xml
index 8066eae..cee45fd 100644
--- a/data/org.kde.kmines.appdata.xml
+++ b/data/org.kde.kmines.appdata.xml
@@ -135,4 +135,7 @@
     <release version="4.0.21121" date="2022-01-06"/>
     <release version="4.0.21120" date="2021-12-09"/>
   </releases>
+  <content_rating type="oars-1.1">
+    <content_attribute id="violence-cartoon">mild</content_attribute>
+  </content_rating>
 </component>
-- 
2.35.1

